documentation/rewrite.asciidoc

```diff
@@ -0,0 +1,35 @@
+:toc:
+toc::[]
+
+= rewrite
```

Comment:
Should we explain this tool more detailed? The Getting Started page gives already a good introduction to the use of openrewrite

Reply:
Your link to the getting started page is fine and should be sufficient. Documentation can always be improved but this is a great starting point.